client/src/components/Login/LoginInterface.js

```diff
@@ -53,7 +53,34 @@ export class LoginInterface extends Component {
       
     if(rules.required){
       isValid = value.trim() !== '' && isValid;
-    } 
+    }
```

Comment:
Do we need validation on login? I'm trying to think of a website that does this, usually its just on signup. So then if requirements for login change, for example if a website increased the minimum length of password, users who signed up before the change would not be able to login.

Reply:
This only checks to see if there is input in both fields so if nothing is inserted then it won't let the user submit. This doesn't need to be implemented but I think this is a nice feature.